pyanaconda/installation.py

```diff
@@ -117,9 +117,10 @@ def _prepare_configuration(storage, payload, ksdata):
 
     # schedule network configuration (if required)
     if conf.system.provides_network_config:
+        overwrite = isinstance(payload, LiveImagePayload)
         network_config = TaskQueue("Network configuration", N_("Writing network configuration"))
         network_config.append(Task("Network configuration",
-                                   ksdata.network.execute, (payload, )))
+                                   network.write_configuration, (overwrite, )))
```

Comment:
We should call `InstallWithTasks` of the Network module here, but it will be easier to implement it after the Payload module is done.